src/main/java/com/orasi/bluesource/AddRoleType.java

```
@@ -0,0 +1,179 @@
+	@FindBy(css = "table.table-responsive.table-bordered") private Webtable rolesTable;
+	@FindBy(xpath = "//*[@id=\'new_role\']/div[1]/div/div") private Element billingLbl;
+	@FindBy(css = "table.table-striped.table-hover") private Webtable projectsTable;
+	@FindBy(xpath = "//*[@id=\"accordion\"]/div/div[13]/div/div[1]/button[1]") private Button newRoleButton;
+	@FindBy(xpath = "//*[@id=\"new_role\"]/div[1]/div/div") private Label lblBilling;
+	@FindBy(id = "role_role_type_id") private Listbox listRoleTypes;
+	@FindBy(xpath = "//input[@value='Create Role']") private Button btnAssignRoleType;
+	
+	private String newRoleTypeName;
+
+	public AddRoleType(OrasiDriver driver){
+		this.driver = driver;
+		ElementFactory.initElements(driver, this);
+	}
+
+	public void openRoleType() {
+
+		if (lnkAdminTab.isDisplayed() == true)
+		{
+			lnkAdminTab.click();
+			lnkRoleTypes.click();
+		}
+	}
+
+	public void click_AddNewRoleTypeButton() {
+
+		if (lnkAddNewRoleTypeButton.isDisplayed() == true)
+		{
+
+			lnkAddNewRoleTypeButton.click();
+		}
+	}
+
+	public void createNewRole(String roleType) {
+		
+		this.newRoleTypeName = roleType;
+
+		if (txtRoleName.isDisplayed()==true) {
+			txtRoleName.sendKeys(newRoleTypeName);
+			billableCheckBox.uncheck();
+
+			Select select = new Select(driver.findElement(By.name("role_type[permission]")));
+			select.selectByValue("View");
+
+			btnCreateRoleType.click();
+		}
+	}
+
+
+	public boolean verifyNewRole() {
+
+		Boolean verify = true;
+
+		if (rolesTable.isDisplayed() == verify) {
+
+			if (rolesTable.getRowWithCellText(newRoleTypeName) > 0) {		
+				System.out.println("New Role Type is created");
+			}
+			else {
+				System.out.println("New role type is not created");
+				verify = false;
+			}
+		}
+
+		return verify;
+	}
+
+	public int getProjectsTableRows() {
+		int rowCount = 0;
+		try
+		{
+			rowCount = projectsTable.getRowCount();
+		}
+		catch(NullPointerException e)
```

You do not want to just swallow this exception...I would rather you  do a sync on the projectsTable element. then get the row count.  
```
projectsTable.syncVisible();
return projectsTable.getRowCount();
```